README.md

```diff
@@ -66,8 +66,9 @@ apply styling, it will operate on the accessibility tree.
     <td>
       ✅ Disabled<br>
       ❌ Hover Behavior Configurable<br>
-      ❌ <a href="https://www.w3.org/TR/wai-aria-1.2/#menuitemcheckbox" target="_blank"><code>role="menuitemcheckbox"</code></a><br>
-      ❌ <a href="https://www.w3.org/TR/wai-aria-1.2/#menuitemradio" target="_blank"><code>role="menuitemradio"</code></a>
+      ✅ <a href="https://www.w3.org/TR/wai-aria-1.2/#menuitem" target="_blank"><code>role="menuitem"</code></a><br>
+      ✅ <a href="https://www.w3.org/TR/wai-aria-1.2/#menuitemcheckbox" target="_blank"><code>role="menuitemcheckbox"</code></a><br>
+      ✅ <a href="https://www.w3.org/TR/wai-aria-1.2/#menuitemradio" target="_blank"><code>role="menuitemradio"</code></a>
```

Comment:
This would receive a ✅ if there is functionality support for this. ie. when a `role="menuitemcheckbox"` will receive `aria-checked="checked"`.

This must be reverted then.

Reply:
okay, thanks. Typescript fixed, and ✅ reverted for role="menuitemcheckbox" and role="menuitemradio"